### PR TITLE
 Add --open option to run command #12233

### PR DESCRIPTION
**Pull request by nameception**
My attempt at adding the --open option for the run command #12233, wasn't able to test it as a option since "run" asks for a meteor app, which i wasn't able to create using the checkout version.

### CODE REVIEW

File: tools/cli/commands.js [hunk 1]
```diff
@@ -406,6 +408,16 @@ function doRunCommand(options) {
     runLog.setRawLogs(true);
   }
 
+  //Opens a browser window when it finishes building
+  if (options.open) {
+    if(process.env.ROOT_URL){
+      open(process.env.ROOT_URL)
+    } else {
+      open('https://localhost:3000')
```
Comment (StorytellerCZ) [inline tools/cli/commands.js:416]: I think the port should be taken from the `port` option and default to 3000 only if none is defined.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

File: tools/cli/commands.js (hunk 1 above)
Comment (nameception) [reply tools/cli/commands.js]: Updated it, if a port is provided it will use it, if not it should default to 3000

File: package.json [hunk 2]
```diff
@@ -29,11 +29,16 @@
...
+            "resolve"
+          ]
         }
       ],
       "no-use-before-define": "warn",
       "import/no-unresolved": "warn"
     }
+  },
+  "dependencies": {
+    "open": "8.4.2"
```
Comment (Grubba27) [inline package.json:42]: this should not be here and there is no need to update package-lock.json

**Comment by Grubba27** [comment]
I made some small adjustments and tested them! It looks great! Thanks a lot for your contribution.

@StorytellerCZ this should go to a dev-bundle branch as it adds a new dependency to our dev-bundle.

**Review by Grubba27 — APPROVED** [review]

**Review by denihs — APPROVED** [review]

**Pull request merged**